### PR TITLE
CONN-1739 added profile to execute sql scripts to maven build

### PR DESCRIPTION
**Pull request by ptambellini01**

### CODE REVIEW

File: Product/Production/Deploy/pom.xml [hunk 1]
```diff
@@ -11,6 +11,14 @@
   <packaging>pom</packaging>
   <modelVersion>4.0.0</modelVersion>
   
+    <properties>
+        <db.host>localhost</db.host>
+        <db.password>nhincpass</db.password>
```
Comment (pvenkatakrishnan1) [inline Product/Production/Deploy/pom.xml]: Is it better to keep the username password in different settings file rather than in the pom?

Comment (ptambellini01) [reply Product/Production/Deploy/pom.xml]: This is already defined in the SoapUI_Test pom. So to be consistent I had to place it here.

File: Product/Production/Deploy/pom.xml [hunk 2]
```diff
@@ -479,7 +487,50 @@
         </plugins>
       </build>
     </profile>
-
+    <profile>
+	     <id>SBScripts</id>
```
Comment (mhpnguyen) [inline Product/Production/Deploy/pom.xml]: maybe use different name for relevant meaning

File: Product/Production/Deploy/pom.xml [hunk 3]
```diff
@@ -11,6 +11,14 @@
   <packaging>pom</packaging>
   <modelVersion>4.0.0</modelVersion>
   
+    <properties>
+        <db.host>localhost</db.host>
```
Comment (TabassumJafri) [inline Product/Production/Deploy/pom.xml]: Define db.host and other db config params in pom at CONNECT folder. Remove it from both SoapUI_Test and Deploy.

File: Product/Production/Deploy/pom.xml [hunk 4]
```diff
@@ -479,7 +487,50 @@
...
+          <plugins>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>sql-maven-plugin</artifactId>
+              <version>1.5</version>
+            <dependencies>
+              <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>5.1.10</version>
```
Comment (mhpnguyen) [inline Product/Production/Deploy/pom.xml]: Define version in connect pom and use place holder to display similar like this<version>${mysqldriver.version}</version>

File: Product/Production/Deploy/pom.xml [hunk 5]
```diff
@@ -479,7 +487,50 @@
...
+            <executions>
+              <execution>
+                <phase>process-test-resources</phase>
+                <goals>
+                  <goal>execute</goal>
+                </goals>
+                <configuration>
+                  <autocommit>true</autocommit>
+                  <srcFiles>
+                    <srcFile>../../../DBScripts/nhincdb/dropall.sql</srcFile>
```
Comment (TabassumJafri) [inline Product/Production/Deploy/pom.xml:515]: Also define path to dbscripts in CONNECT pom.xml

**Review by TabassumJafri — CHANGES_REQUESTED** [review]

File: Product/Production/Deploy/pom.xml (hunk 5 above)
Comment (ptambellini01) [reply Product/Production/Deploy/pom.xml:515]: Cannot establish an absolute path. Will go ahead with the present path assignment

**Review by TabassumJafri — APPROVED** [review]

**Review by mhpnguyen — APPROVED** [review]

**Pull request merged**